src/HasSuffixCollection.php

```
@@ -127,120 +103,111 @@ public function getSuffixTag(): BlockInterface|false|InlineInterface|VoidInterfa
+     * $component->suffixClass('override-class', true);
      * ```
      *
-     * @param string|Stringable ...$values Suffix content to set for the element.
+     * @param string|Stringable|UnitEnum $value CSS class name to add.
+     * @param bool $override Whether to override existing class value.
      *
-     * @return static New instance with the updated `suffix` value.
+     * @return static New instance with the updated `suffixAttributes` value.
      */
-    public function suffix(Stringable|string ...$values): static
+    public function suffixClass(string|Stringable|UnitEnum $value, bool $override = false): static
```

PHPMD (2.15.0)

[error] The method suffixClass has a boolean flag argument $override, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)